### PR TITLE
[Tensor-IF] An anchor for further development.

### PR DESCRIPTION
**Pull request by myungjoo**
Anyone can jump in and start the development!

This is for #1703 

Signed-off-by: MyungJoo Ham <myungjoo.ham@samsung.com>

ps. I'm going to add test cases before the feature implementation for TDD.

### CODE REVIEW

File: gst/nnstreamer/tensor_if/gsttensorif.h [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+
+/**
+ * @brief Behaviors that may fit in THEN and ELSE
+ * @details FILL_WITH_FILE, FILL_WITH_FILE_RPT, and REPEAT_PREVIOUS_FRAME caches an output frame
+ *          and thus, may consume additional memory and incur an additional memcpy.
+ */
+typedef enum {
+  TIFB_PASSTHROUGH = 0,	/**< The input frame becomes the output frame */
+  TIFB_SKIP,	/**< Do not generate output frame (frame skip) */
+  TIFB_ZERO_VALUES,	/**< Fill 0 for the output frame */
```
Comment (dongju-chae) [inline gst/nnstreamer/tensor_if/gsttensorif.h:111]: How about `TIFB_FILL_ZEROS` ?

Comment (myungjoo) [reply gst/nnstreamer/tensor_if/gsttensorif.h:111]: That looks better! Thanks.

File: gst/nnstreamer/tensor_if/gsttensorif.h [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+ *           operand=EQ
+ *           supplied_values=1
+ *           then=PASSTHROUGH # or whatsoever you want
+ *           else=SKIP # or whatsoever you want
+ *    !
+ * tensor_demux name=demux !
+ *      demux.src_0 ! queue ! fakesink # throw away (we don't need the 1/0 value in the end)
+ *      demux.src_1 ! queue ! do whatever you want
+ * ...
+ */
```
Comment (jijoongmoon) [inline gst/nnstreamer/tensor_if/gsttensorif.h:67]: Looks good to me. we could make more complicated pipeline with this!

**Comment by myungjoo** [comment]
I'll continue this. Once this becomes "buildable with empty function definitions", I'll hand this over to any volunteer.

**Pull request merged**